dataset/dictionary/z-archives.json5

```
@@ -932,73 +1061,85 @@
   {
     en: "Reign of Violet",
     ja: "菫染の国",
+    zhCN: "堇染之国",
     notes: "菫色ノ庭の挑戦名",
     tags: [ "inazuma" ],
   },
   {
     en: "Thundering Valley",
-    ja: "初雪の幽谷",
+    ja: "初雪の幽谷", // Maybe it should be 初雷 rather than 初雪？
```

Oops, yes, it should be 初雷.

```suggestion
    ja: "初雷の幽谷",
```